app/main.py

```diff
@@ -1,7 +1,23 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+
+def get_weather(api_key: str, city: str = "Paris") -> None:
```

Comment:
The city name 'Paris' should be defined as a constant in uppercase style, as per the previous review comments. This will improve the readability and maintainability of your code.